CHANGES/7974.feature

```diff
@@ -0,0 +1 @@
+Added pagination to the _catalog and the tags endpoint in the registry API.
```

Comment:
s/tags/tags/list

Reply:
```suggestion
Added pagination to the _catalog and the tags/list endpoint in the registry API.
```
Ha, i needed to read you sed-ish twice...